profile.graphqls

```diff
@@ -39,6 +39,22 @@ type ProfileTaskCreationResult {
     id: String
 }
 
+# Profile task execute log
+type ProfileTaskLog {
+    id: String!
+    # profile task id
+    taskId: String!
+    # execute instance id
+    instanceId: ID!
+    # operation type
+    # 1: when sniffer has notified
+    # 2: when sniffer has execute finish to report
+    operationType: Int!
```

Comment:
Are you expecting two log records, one for notification, the other for finished? As there is no update for this entity in the backend.

Reply:
Yes, there will have two log records. No update operation on the database.

Reply:
operationType should be a type, rather than an int.

Reply:
Changed that. Please have a look is that ok.